README.md

```diff
@@ -30,6 +30,16 @@ http://stackoverflow.com/questions/49002/prefer-composition-over-inheritance
 - Jim Grant
 - Johnson Jew
 
+## bst
+###Collaborators
+- Jim Grant
+- Megan Slater
+
+a binary serach tree that contains methods to determine size, balance, and depth
+as well as methods to insert a new value and determine whether any particular
+value is contained within the tree.  Also has in order, pre order, post order and breadth first
+traversal methods.  Inspiration for depth and balance methods taken from [Jonathan Stallings](https://github.com/jonathanstallings/data-structures/blob/feature/bst/bst.py).
+
```

Comment:
This should be documented thoroughly. From the assignment description: _Add documentation of your new methods to your README file._